### ISSUE TITLE
About using all data when fine-tuning

### ISSUE THREAD

**Issue by kimwongyuda** [JiachengLi1995/UCTopic#3]
When reading your paper, I wondered the below question.

In Finetuning Setup of section 4.2. Entity Clustering, there is a sentence "Because UCTOPIC is an unsupervised method, we use all data to finetune and evaluate.".

Even though UCTOPIC is a unsupervised model, is it inappropriate to use training + test(valid) data for training?
In other or related papers, are they also using such way?

Thank you!

**Comment by JiachengLi1995** [comment]
Thanks for your interest in our project!
Because we do not use any label in the test or valid data, we believe it's okay to use all data for training.
We use all data for all baselines for a fair comparison.
I think even if UCTopic is trained on only training data, our model can still achieve the best result.

**Comment by kimwongyuda** [comment]
Thank you so much for replying !!!
